### ISSUE TITLE
prepend for 3.x fix

### ISSUE THREAD

**Issue by linkingarts** [CakePHP-Bootstrap/cakephp2-bootstrap2-helpers#2]
changed lines 210-212

```
        $beforeClass .= ' input-group input-prepend' ;
        if (is_string($prepend)) {
            $before .= '<span class="input-group-addon">'.$prepend.'</span>' ;
```

did not need append so I didn't test that yet

**Comment by Holt59** [comment]
Thanks, I updated the file.

**Comment by linkingarts** [comment]
awesome

Not sure you have time for this but it would be great if you could separate
the 2.x and 3.x files so the repo could be added as a submodule

cheers,

_peter

On Mon, Oct 28, 2013 at 2:22 PM, Mikaël Capelle notifications@github.comwrote:

> Closed #2 https://github.com/Holt59/cakephp-bootstrap-helpers/issues/2.
> 
> —
> Reply to this email directly or view it on GitHubhttps://github.com/Holt59/cakephp-bootstrap-helpers/issues/2
> .

## 

Peter Jacobson
Linking Arts
www.linkingarts.com
www.wedocracy.com
www.petejacobson.com
twitter.com/petejacobson
twitter.com/linkingarts
+1 347.223.5466 / mobile
peterjacobson / Skype

people + technology + art
